devops/db/liquibase/changelogs/payment-transaction-gateway/scripts/table-creation/V7__PM_988_ADD_COLUMN_STATUS.sql

```diff
@@ -0,0 +1,2 @@
+-- create STATUS VARCHAR2 column
+ALTER TABLE PP_PGS_REQUEST_INFO ADD STATUS VARCHAR2(20);
```

Comment:
@andrearovere I changed the column size to 20 since for the moment we are storing only values no longer than 10 characters